chart/templates/ph-app-templates/mlflow.yaml

```diff
@@ -0,0 +1,57 @@
+  defaultEnvs:
+  - name: BACKEND_STORE_URI
+    description: "URI to which to persist experiment and run data. Acceptable URIs are SQLAlchemy-compatible database connection strings (e.g. ‘sqlite:///path/to/file.db’) or local filesystem URIs."
+    defaultValue: "sqlite:///$(PRIMEHUB_APP_ROOT)/mlflow.db"
+    optional: false
+  - name: DEFAULT_ARTIFACT_ROOT
+    description: "Local or S3 URI to store artifacts, for new experiments."
+    defaultValue: "$(PRIMEHUB_APP_ROOT)/mlruns"
+    optional: false
+  - name: STATIC_PREFIX
```

Comment:
The static prefix is not required to be configurable variable. Just add it to the args filed in the container spec.